plugin/__init__.py

```diff
@@ -0,0 +1,39 @@
+from pyramid.events import NewRequest
+from kinto.core.events import AfterResourceChanged
+from pywebpush import WebPusher
+
+def on_new_request(event):
+    cache = event.request.registry.cache
+    storage = event.request.registry.storage
+
+    subsID = req.param('subsID')
+    pushURL = req.param('pushURL')
+    trigger = req.param('trigger')
+
+    value = [pushURL, trigger]
+    cache.set(subsID, value)
+
+    # Not sure about this
+    entry = storage.create(collection_id='subscriptioninfo',
+                           record=value)
+
+config.add_subscriber(on_new_request, NewRequest)
+
+
+
+def on_resource_changed(event)
+    action = payload['action']
```

Comment:
We would rather use a Listener to be able to configure which buckets/collections the plugin will be activated.